client/src/components/public/calendar/EventTypeDetails.vue

```diff
@@ -83,7 +84,12 @@ export default {
         ...mapGetters('publicEvent', {
             eventType: getters.GET_EVENT_TYPE,
             publicEvent: getters.GET_PUBLIC_EVENT
-        })
+        }),
+        startDateFormatted() {
+            return moment(this.publicEvent.startDate).format(
+                'dddd, YYYY-MM-DD, HH:mm'
```

Comment:
HH:mm:ss

Reply:
This date shouldn't contain seconds, I think it would be better if there were only hours and minutes, like on Calendly